nerfstudio/models/nerfacto.py

```diff
@@ -283,7 +283,16 @@ def get_outputs(self, ray_bundle: RayBundle):
         weights_list.append(weights)
         ray_samples_list.append(ray_samples)
 
-        rgb = self.renderer_rgb(rgb=field_outputs[FieldHeadNames.RGB], weights=weights)
+        background_color = (
+            self.renderer_rgb.get_background_color(
+                rgb=field_outputs[FieldHeadNames.RGB], background_color=self.config.background_color
+            )
+            if self.training
+            else None
```

Comment:
Using `else "last_sample"` instead of `else None` fixes the mesh export quality. I can't think of a neat fix apart from re-introducing the `use_alpha_transparency_training` flag. Any ideas?

Reply:
With the change:
![with_change01](https://github.com/nerfstudio-project/nerfstudio/assets/53228351/4c2b4fae-0cfb-47dd-a323-9b3671317e66)

Without change:
![without_change02](https://github.com/nerfstudio-project/nerfstudio/assets/53228351/aa2071b3-1028-42ba-8080-b60dfde76d7b)